src/extensions/default/CodeFolding/main.js

```diff
@@ -270,9 +270,12 @@ define(function (require, exports, module) {
 
         // Reuse any existing fold gutter
         if (gutters.indexOf(GUTTER_NAME) < 0) {
-            var lnIndex = gutters.indexOf("CodeMirror-linenumbers");
+            var lineNumberIndex = gutters.indexOf("CodeMirror-linenumbers");
+            if (lineNumberIndex < 0) {
+                $(editor.getRootElement()).addClass("linenumber-disabled");
```

Comment:
This might also go well here: https://github.com/adobe/brackets/blob/7814da193b98ff7889714a601d630973bddf02d0/src/editor/Editor.js#L2324-L2328

That way, stylesheets and other extension have an easy way to reuse this state.
cc @nethip